datastore/query.go

```diff
@@ -171,25 +177,46 @@ func (q *Query) Transaction(t *Transaction) *Query {
 }
 
 // Filter returns a derivative query with a field-based filter.
+//
+// DEPRECATED. Use the FilterWithArgs() method instead, which supports the same
+// set of operations (and more).
```

Comment:
```suggestion
// Deprecated: Use the FilterField method instead, which supports the same
// set of operations (and more).
```

https://github.com/golang/go/wiki/Deprecated